Gopkg.toml

```diff
@@ -44,8 +44,9 @@
   branch = "release-4.0"
 
 [[constraint]]
-  branch = "master"
+  branch = "PTX-1620"
```

Comment:
first merge https://github.com/portworx/sched-ops/pull/134#pullrequestreview-259120232 and then revert this back to master